src/DoctrineORMModule/Service/ConfigurationFactory.php

```diff
@@ -152,6 +152,10 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             $config->setSecondLevelCacheEnabled();
             $config->setSecondLevelCacheConfiguration($cacheConfiguration);
         }
+        
+        if($schemaFilter = $options->getSchemaFilter()) {
```

Comment:
Why are you using `getSchemaFilter` when this passes to the function `setFilterSchemaAssetsExpression`?  It seems to me we should keep the `setFilterSchemaAssetsExpression` name for the `Configuration`.